gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/FsJobStatusRetrieverTest.java

```diff
@@ -51,6 +51,7 @@ public void setUp() throws Exception {
         ConfigValueFactory.fromAnyRef(stateStoreDir));
     this.jobStatusRetriever = new FsJobStatusRetriever(config, mock(MultiContextIssueRepository.class));
     this.mysqlDagActionStore = mock(MysqlDagActionStore.class);
+    // job status monitor stores dag action into dag action store, so need to mock that behavior to avoid NPE
```

Comment:
I did initially suggest adding such a comment, but as I just mentioned, my opinion has evolved.  the need for mocking here seems a code smell that `KJSM::addJobStatusToStateStore` is doing too much and ought to be reworked.  following that, no mocking and no comment would be required